### PR TITLE
QT5 link is fixed 

### PR DESCRIPTION
**Pull request by strikersix23**
Compilation now works on windows msvc

### CODE REVIEW

File: CI/before_script.msvc.sh [hunk 1]
```diff
@@ -421,7 +421,7 @@ if [ -z $SKIP_DOWNLOAD ]; then
 		fi
 
 		download "Qt 5.7.0" \
-			"https://download.qt.io/archive/qt/5.7/5.7.0/qt-opensource-windows-x86-msvc${MSVC_YEAR}${QT_SUFFIX}-5.7.0.exe" \
+			"http://download.qt.io/new_archive/qt/5.7/5.7.0/qt-opensource-windows-x86-msvc${MSVC_YEAR}${QT_SUFFIX}-5.7.0.exe" \
```
Comment (akortunov) [inline CI/before_script.msvc.sh]: Why do you use http instead of https?

Comment (psi29a) [reply CI/before_script.msvc.sh]: Agreed, if possible, switch back to https please.

Comment (lostmsu) [reply CI/before_script.msvc.sh]: Tested, works with https.

Comment (akortunov) [reply CI/before_script.msvc.sh]: Fixed upstream to do not waste a time for rebasing.

**Pull request closed without merging**